.azure-pipelines/azure-pipelines-backend.yml

```diff
@@ -0,0 +1,50 @@
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    exclude: 
+      - /frontend
+      - /.azure-pipelines/azure-pipelines-frontend.yml
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps: 
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '12.17.0'
+    displayName: 'Install Node.js 12.17.0'
+
+  - script: |
+      yarn global add @angular/cli@8.3.21
+      yarn global add @nestjs/cli@6.12.6
+      yarn global add typescript@3.7.2 
```

Comment:
Sorry about this! Check the README for updated versions of Angular, Nest, and TypeScript.

Reply:
Updated. Also removed Angular, as the backend doesn't use it.